driver-mapping/src/main/java/com/datastax/driver/mapping/MappingConfiguration.java

```diff
@@ -17,11 +17,6 @@
 
 /**
  * The configuration to use for the mappers.
- * <p/>
- * The following parts can be configured:
- * <ul>
- * <li>{@link PropertyMapper}: the mapper component that maps Java properties to table columns and UDT fields.</li>
- * </ul>
```

Comment:
This is a text description of the builder methods, I think the API is obvious enough that we can remove it, and avoid having to keep it in sync.